include/cutlass/gemm/collective/intel_pvc_mma.hpp

```diff
@@ -184,72 +195,79 @@ struct CollectiveMma<
     (void)thread_idx;
     (void)smem_buf;
 
-    static_assert(is_rmem<FrgTensorD>::value, "D tensor must be rmem resident.");
-    static_assert(is_tuple<typename TensorA::engine_type::iterator::value_type>::value, "A tensor must be a tuple iterator.");
-    static_assert(is_tuple<typename TensorB::engine_type::iterator::value_type>::value, "B tensor must be a tuple iterator.");
-    static_assert(is_rmem<FrgTensorC>::value, "C tensor must be rmem resident.");
```

Comment:
Why removing the asserts?

Reply:
because TensorA and TensorB are not TMA tensor any more, line 187 and line 190 will be reverted